themes/classic/_dev/js/product.js

```diff
@@ -104,7 +104,7 @@ $(document).ready(function () {
       max: 1000000
     });
 
-    quantityInput.on('change', function (event) {
+    quantityInput.on('keyup change', function (event) {
```

Comment:
This will trigger the event only if you type or you modify the value through touchspin ("change" does not trigger event when switching focus out of the field)